bindata/network/multus/multus.yaml

```diff
@@ -185,12 +185,15 @@ spec:
         - name: system-cni-dir
           hostPath:
             path: {{ .SystemCNIConfDir }}
+            type: Directory
```

Comment:
@dougbtv is DirectoryOrCreate useful here, or do we need specific permissions on the directory? IIRC DirectoryOrCreate will use 755 and kubelet's user/group which may not be what we want, so maybe it's not useful.

Reply:
Agreed, MCO owns these directories, so DirectoryOrCreate is not the right choice here

Reply:
Oh hah, yeah, I'll be containernetworking-cni RPMs create the dir, or maybe CRIO RPMs do. Fun times.